components/list/list-item-drag-drop-mixin.js

```diff
@@ -330,6 +330,9 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 			:host([_drop-location="7"]) [slot="control-container"]::after {
 				border-color: transparent;
 			}
+			:host(:not([no-primary-action])) [slot="outside-control-action"] ~ [slot="content"] {
```

Comment:
Removing this from `ListItemLinkMixin` had an impact on drag and drop css, which I discovered was this.

To do:
- [ ] Test out scenario where interactive element is in drag and drop item